src/OrbitGl/ThreadTrack.cpp

```diff
@@ -245,7 +244,7 @@ Color ThreadTrack::GetTimerColor(const TimerInfo& timer_info, bool is_selected,
 
 bool ThreadTrack::IsEmpty() const {
   return thread_state_bar_->IsEmpty() && event_bar_->IsEmpty() && tracepoint_bar_->IsEmpty() &&
-         TimerTrack::IsEmpty();
```

Comment:
First of all, this was introduced 4 days ago by (https://github.com/google/orbit/pull/2961). We were using before timer_data_ and I wanted to explicitly say we were using TimerTrack here (because timer_data_ was a member of TimerTracks), before replacing it by the provider query.

Second, yes. I could try to erase that inheritance. Makes sense for me. But I repeat the same that I said in the other PR, the optimization didn't make any change in the inheritance.

Reply:
Ah right, I thought this was like this way before.
Regarding removing the inheritance, I don't think this should be done in the same PR anyways, and I'm not 100% sure it makes sense TBH.

Reply:
Talking offline, we agree that removing the inheritance is not a priority but we should at least make a bug.